snuba/web/views.py

```diff
@@ -72,126 +56,27 @@
 from snuba.subscriptions.codecs import SubscriptionDataCodec
 from snuba.subscriptions.data import PartitionId
 from snuba.subscriptions.subscription import SubscriptionCreator, SubscriptionDeleter
+from snuba.utils.health_info import (
+    IS_SHUTTING_DOWN,
```

Comment:
I think this is going to cause a problem due to how python imports work. Do you know how this variable is used? Can you think of some ways that this code will change in functionality now that this variable has been exported somewhere else?

Reply:
Made `get_shutdown` and `_set_shutdown`